### PR TITLE
Bug fixes, better page transitions, updated tab functionality and term placeholders, adds legit content

### PR DESCRIPTION
**Pull request by himedlooff**

### CODE REVIEW

**Comment by himedlooff** [comment]
Can I get a review and merge please? 

selfie-1

**Comment by mthibos** [comment]
What on earth?

File: src/form-explainer/closing-disclosure.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+        <div class="oah-title-tag">
+          <a href="/owning-a-home/" class="go-back-link l-back-link">Owning a Home</a>
+        </div>
+
+        <div class="page-intro">
+          <h1 class="page-title">Closing Disclosure Explainer</h1>
+          <div class="wrapper">
+            <div class="col-9">
+              <p class="lead">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Ducimus illum quae aliquid quis vero saepe soluta pariatur harum alias ab odio facilis autem temporibus? Laudantium natus nulla possimus amet qui.</p>
```
Comment (ascott1) [inline src/form-explainer/closing-disclosure.html]: Is there text that could be used here in place of the ipsum?

Comment (himedlooff) [inline src/form-explainer/closing-disclosure.html]: Good question. Looking into it.

File: src/static/js/modules/form-explainer.js [hunk 2]
```diff
@@ -75,112 +75,182 @@ function updateStickiness() {
...
+ * @return {null}
+ */
+function setCategoryPlaceholders( id ) {
+  var $currentPage = $WRAPPER.find( '#explain_page-' + id ),
+      categories = [ 'checklist', 'alerts', 'definitions' ];
+  for ( var i = 0; i < categories.length; i++ ) {
+    var category = categories[i];
+    if ( $currentPage.find( '.expandable__form-explainer-' + category ).length === 0 ) {
+      placeholderHTML = '' +
+      '<div class="expandable expandable__padded expandable__form-explainer ' +
```
Comment (ascott1) [inline src/static/js/modules/form-explainer.js]: This could all be moved into a handlebars template to separate the html from the js

Comment (himedlooff) [inline src/static/js/modules/form-explainer.js]: Cool. Is there a good example to follow to get me started?

File: src/static/js/modules/form-explainer.js [hunk 3]
```diff
@@ -75,112 +75,182 @@ function updateStickiness() {
...
+      $WRAPPER.find('.expandable__form-explainer-placeholder').hide();
     } else {
-      getCurrentPage().find('.expandable__form-explainer').hide();
-      getCurrentPage().find( '.expandable__form-explainer-' + target ).show();
-      getCurrentPage().find('.image-map_overlay').hide();
-      getCurrentPage().find( '.image-map_overlay__' + target ).show();
+      $WRAPPER.find('.expandable__form-explainer').hide();
+      $WRAPPER.find( '.expandable__form-explainer-' + target ).show();
+      $WRAPPER.find('.image-map_overlay').hide();
+      $WRAPPER.find( '.image-map_overlay__' + target ).show();
```
Comment (ascott1) [inline src/static/js/modules/form-explainer.js]: Could we store the DOM selections in variables to cache them?

Comment (ascott1) [inline src/static/js/modules/form-explainer.js]: If not we could create show/hide functions that make this a bit DRYer

File: src/static/js/modules/form-explainer.js (hunk 2 above)
Comment (ascott1) [inline src/static/js/modules/form-explainer.js]: They're used a lot in `check-rates.js.` I'm happy to go over how to get it started here.

**Comment by ascott1** [comment]
Left some comments, but overall this looks great. Nicely done @himedlooff

**Comment by ascott1** [comment]
selfie-1

File: src/static/js/modules/form-explainer.js [hunk 4]
```diff
@@ -75,112 +75,182 @@ function updateStickiness() {
...
+    if ( $WINDOW.width() >= 600 ) {
+      $imageMapWrapper.sticky({ topSpacing: 30 });
+    }
   }
+  setCategoryPlaceholders( id );
+}
+
+/**
+ * Weird hack to get sticky() to update properly.
+ * We're basically jinggling the window to force a sticky() repaint.
```
Comment (cfarm) [inline src/static/js/modules/form-explainer.js]: this is awesome

Comment (himedlooff) [inline src/static/js/modules/form-explainer.js]: lulz

also my typo is awesome "jinggling"? i might have been losing my mind at this point

**Pull request merged**